tests/generate_source/lib.c.txt

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Not required, but you could embed this file in `gen.sh` using a heredoc: http://tldp.org/LDP/abs/html/here-docs.html

```shell
cat >$c_src <<EOF
#include <stdio.h>
#include "foo/foo.h"

void foo() {
    printf("foo()\n");
}
EOF
```

Reply:
*and lib.h, too.

Reply:
I don't want such solution because we need to add this to gitignore

Reply:
(i.e. generate directly in the build directory directly from the script, so we don't have as  many source files containing ~5 lines of C :) )

Reply:
Still such solution isn't nice for me. Other tests that we have use same approach. Just prefer to keep template files rather than template in source code.

Reply:
OK, sure!